yt/frontends/enzo/fields.py

```diff
@@ -65,6 +84,12 @@ class EnzoFieldInfo(FieldInfoContainer):
     )
 
     def __init__(self, ds, field_list):
+        # Data stored as a list of dictionaries
```

Comment:
So I was thinking we'd do all the file loading in the creation of aliases, but I now am not sure which I prefer.

Reply:
Oh, I see. I didn't even think of that, to be honest. That might be better.